packages/card/src/__demo__/examples/crazyOrder.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import Button from '@mineral-ui/button';
+import Card from '../../Card';
+import CardActions from '../../CardActions';
+import CardBlock from '../../CardBlock';
+import CardTitle from '../../CardTitle';
+import CardImage from '../../CardImage';
+
+const Root = createStyledComponent('div', {
+  alignItems: 'flex-start',
+  display: 'flex',
+
+  '& > *': {
+    flex: '1 1 25%'
+  },
+
+  '& > *:not(:first-child)': {
+    marginLeft: '2em'
+  }
+});
+
+function Example() {
+  return (
+    <Root>
+      <Card>
+        <CardImage src="http://fillmurray.com/1000/563" alt="Bill Murray" />
```

Comment:
Wow, fillmurray.com is pretty awesome.  It really adds a touch of class to your examples.  We should however have a follow-up story to replace these images with mineral related placeholders once we get some design assets.